examples/gallery/lines/linestrings.py

```diff
@@ -0,0 +1,39 @@
+"""
+Plotting lines with LineString/MultiLineString geometry
+=======================================================
+
+The :meth:`pygmt.Figure.plot` method allows us to plot geographical data such
+as lines which are stored in a :class:`geopandas.GeoDataFrame` object. Use
+:func:`geopandas.read_file` to load data from any supported OGR format such as
+a shapefile (.shp), GeoJSON (.geojson), geopackage (.gpkg), etc. Then, pass the
+:class:`geopandas.GeoDataFrame` as an argument to the ``data`` parameter of
+:meth:`pygmt.Figure.plot`, and style the geometry using the ``pen`` parameter.
+"""
+
+# %%
+import geodatasets
+import geopandas as gpd
+import pygmt
+
+# Read shapefile data of large rivers in Europe using geopandas
+gdf = gpd.read_file(geodatasets.get_path("eea large_rivers"))
+
+# Convert object to EPSG 4326 coordinate system
+gpd = gpd.to_crs("EPSG:4326")
+
+fig = pygmt.Figure()
+
+fig.coast(
+    projection="M10c",
+    region=[-10, 30, 35, 57],
+    resolution="l",
+    land="gray95",
+    shorelines="1/0.1p,gray50",
+    borders="1/0.1,gray30",
+    frame=True,
+)
+
+# Add rivers to map
+fig.plot(data=gpd, pen="1p,steelblue")
```

Comment:
For me, a thinner line looks better.
```suggestion
fig.plot(data=gdf, pen="0.5p,steelblue")
```

Reply:
I would prefer 1p since it's the element on which the focus lies in this example.

Reply:
Hm. OK. Edited the suggestion to only include the fix from "gpd" to "gdf".